src/helm/benchmark/scenarios/opinions_qa_scenario.py

```diff
@@ -0,0 +1,172 @@
+    )
+    CODALAB_BUNDLE: str = "0xa6f81cc62d7d4ccb93031a72d2043669"
+    FILE_NAME: str = "Pew_American_Trends_Panel_W{wave}.csv"
+    PEW_SURVEY_WAVES: list = [26, 27, 29, 32, 34, 36, 41, 42, 43, 45, 49, 50, 54, 82, 92]
+
+    def __init__(self, survey_type: str, context: str):
+        super().__init__()
+        assert context in ["default", "steer-qa", "steer-bio", "steer-portray"]
+
+        self.survey_type: str = survey_type
+        self.context: str = context
+
+    def download_data(self):
+
+        self.output_path: str = os.path.join(self.output_path, "data")
+        if not os.path.exists(self.output_path):
+            os.makedirs(self.output_path)
+
+        DOWNLOAD_FILENAMES = [self.FILE_NAME.format(wave=wave) for wave in self.PEW_SURVEY_WAVES]
+        DOWNLOAD_FILENAMES += [f"{steer}.csv" for steer in ["steer-qa", "steer-bio", "steer-portray"]]
+        DOWNLOAD_FILENAMES += ["Pew_American_Trends_Panel_disagreement_500.csv"]
+
+        for filename in DOWNLOAD_FILENAMES:
+            data_path: str = os.path.join(self.output_path, filename)
+
+            source_url: str = self.CODALAB_URI_TEMPLATE.format(bundle=self.CODALAB_BUNDLE, filename=filename)
+            ensure_file_downloaded(source_url=source_url, target_path=data_path, downloader_executable="gdown")
+
+    def read_survey_questions(self, csv_path):
+        df = pd.read_csv(csv_path, sep="\t")
+        df["options"] = df.apply(lambda x: eval(x["options"]), axis=1)
+        return df
+
+    def get_instances(self) -> List[Instance]:
+        self.download_data()
+
+        # Read all the instances
+        instances: List[Instance] = []
+        splits: Dict[str, str] = {
+            "dev": TRAIN_SPLIT,
+            "test": TEST_SPLIT,
+        }
+
+        all_splits = ["dev", "test"] if self.context == "steer-qa" else ["test"]
+        csv_dict = {
+            "dev": os.path.join(self.output_path, f"{self.context}.csv"),
+            "test": os.path.join(self.output_path, f"{self.survey_type}.csv"),
+        }
+
+        bios_df = None
+        if self.context in ["steer-bio", "steer-portray"]:
+            bios_path = os.path.join(self.output_path, f"{self.context}.csv")
+            bios_df = pd.read_csv(bios_path, sep="\t")
+
+        for split in all_splits:
+
+            csv_path: str = csv_dict[split]
+            assert os.path.exists(csv_path)
+
+            question_df = self.read_survey_questions(csv_path)
+
+            for qidx, (question, answers) in enumerate(zip(question_df["question"], question_df["options"])):
+
+                # Opinions QA test questions have no correct answer and thus we set it to be None by default
+                # for all test instances.
+                # In the case where context = steer-qa, we add demographic information in the form of a
+                # in-context question answer pair as shown in the example above.
+
+                correct_answer = None if split == "test" else question_df["correct"][qidx]
```

Comment:
Why is there a correct answer in training?  Is this used?

Reply:
Because this answer is used to provide demographic group information in context. For instance, if we want to steer the model towards Democrats, we include in context the following QA pair:

        Question: In politics today, do you consider yourself a
        A. Republican
        B. Democrat
        C. Independent
        D. Something else
        E. Refused
        Answer: B
        
    Here, "Democrat" is the "correct answer"